pypeit/images/combineimage.py

```diff
@@ -213,6 +213,8 @@ def run(self, bias=None, flatimages=None, ignore_saturation=False, sigma_clip=Tr
                 exptime = np.zeros(self.nfiles, dtype=float)
 
             # Save the lamp status
+            # TODO: As far as I can tell, this is the *only* place rawheadlist
```

Comment:
paging @rcooke-ast or @jhennawi who probably wrote this code.

Reply:
Yes - I think I wrote this. It is only there to warn a user when they combine arc frames with different lamps on. It doesn't serve any other purpose  :-)

Reply:
Leaving it for now.